locale/tr_TR.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -0,0 +1,515 @@
+# PalMA translation.
+# Copyright (C) 2017 Universitätsbibliothek Mannheim
+# This file is distributed under the same license as the PalMA package.
+# Stefan Weil <stefan.weil@bib.uni-mannheim.de>, 2014.
+# Dennis Müller <dennis.mueller@bib.uni-mannheim.de>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PalMA 0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-11-19 20:33+0100\n"
+"PO-Revision-Date: 2017-09-12 14:38+0200\n"
+"Last-Translator: Dennis Müller <dennis.mueller@bib.uni-mannheim.de>\n"
+"Language-Team: English <en@li.org>\n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: index.php:181
+msgid "not available"
+msgstr ""
+
+#: index.php:223
+#, fuzzy
+msgid "Enter valid URL"
+msgstr "Please enter URL"
+
+#: index.php:563
+msgid "You were disconnected!"
+msgstr ""
```

Comment:
`msgstr` should give the Turkish text for all messages. Without that there will not be a Turkish user interface.